src/docfx/config/ops/OpsAccessor.cs

```diff
@@ -23,17 +22,10 @@ internal class OpsAccessor : ILearnServiceAccessor
         private delegate Task<HttpResponseMessage> HttpMiddleware(HttpRequestMessage request, Func<HttpRequestMessage, Task<HttpResponseMessage>> next);
 
         public static readonly DocsEnvironment DocsEnvironment = GetDocsEnvironment();
-
-        private static readonly SecretClient s_secretClientPublic = new(new(
-            Environment.GetEnvironmentVariable("DOCS_KV_PROD_ENDPOINT") ?? "https://docfxdevkvpub.vault.azure.net/"), new DefaultAzureCredential());
-
-        private static readonly SecretClient s_secretClientPubDev = new(new("https://docfxdevkvpubdev.vault.azure.net/"), new DefaultAzureCredential());
-
-        private static readonly SecretClient s_secretClientPerf = new(new("https://kv-opbuild-test.vault.azure.net/"), new DefaultAzureCredential());
-
-        private static readonly Lazy<Task<string>> s_opsTokenPublic = new(() => GetSecret("OpsBuildUserToken", DocsEnvironment.Prod));
-        private static readonly Lazy<Task<string>> s_opsTokenPubDev = new(() => GetSecret("OpsBuildUserToken", DocsEnvironment.PPE));
-        private static readonly Lazy<Task<string>> s_opsTokenPerf = new(() => GetSecret("OpsBuildUserToken", DocsEnvironment.Perf));
+        private static readonly CachedTokenCredential s_cachedTokenCredential = new(new DefaultAzureCredential());
+        private static readonly string s_opsClientId = DocsEnvironment == DocsEnvironment.Prod
+                ? "6befca88-4c28-430a-957e-f870b267bcfc"
+                : "6ce33073-a071-4cf9-9936-f5a24d21a089";
```

Comment:
Is perf using the same id as ppe?

Reply:
Only two aad apps:
![image](https://user-images.githubusercontent.com/42199316/128798520-6e0d7dca-f697-49bd-8a61-dcbaa4c29d0a.png)